packages/std/src/addresses.rs

```diff
@@ -79,6 +85,27 @@ impl CanonicalAddr {
     pub fn is_empty(&self) -> bool {
         self.0.is_empty()
     }
+
+    /// Converts a `CanonicalAddr` into a vector of bytes.
+    ///
+    /// This consumes the `CanonicalAddr`, so we do not need to copy its contents.
+    /// It is equivalent to both `Vec::<u8>::from(addr)` and `let v: Vec<u8> = addr.into()` and just a matter of taste which one you use.
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// # use cosmwasm_std::CanonicalAddr;
+    /// let address = CanonicalAddr::from(vec![0, 187, 61, 11, 250, 0]);
+    /// let bytes = address.into_vec();
+    ///
+    /// assert_eq!(bytes, &[0, 187, 61, 11, 250, 0]);
+    /// ```
+    #[inline]
```

Comment:
nice to ensure zero-cost

Reply:
Oh, I copied this from the String implementation by accident. Generally I think it is better to not try to be more clever than the compiler, which should inline stuff like this anyways. Will check again and probably remove this attribute.